### DISCUSSION TITLE
Authentication, Authorization and Auditing

### DISCUSSION THREAD

**Discussion by sebplorenz** [FraunhoferISST/IDS-ConfigurationManager#124] [Ideas]
Hi,
I think there is an urgent need for implementing authentication, authorization and auditing in the CM and its UI.
Regarding authorization there should be at least two roles. One role for viewing only and another role (like admin) that can change settings.
Users should be able to log in with their credentials. An admin should be able to create users and other admins in the CM. Additionally it should be possible to use identities from existing identity providers or user directories (AD or LDAP).
There should also be an audit trail to see who changed what and when. Since changing the settings of a connector can have a significant impact on the dataspace.
What is the roadmap for the functionalities mentioned above?
Regards, Sebastian

**Comment by ghost** [comment]
I will forward this internally, but I also think that there is a need for the topics addressed above.

**Comment by ghost** [comment]
Status-Update
Is currently being implemented: https://github.com/International-Data-Spaces-Association/IDS-ConfigurationManager/issues/145
